hs_core/irods.py

```diff
@@ -14,10 +14,18 @@ class ResourceIRODSMixin(models.Model):
     class Meta:
         abstract = True
 
-    def __home_path(self):
+    @property
+    def irods_home_path(self):
         """ Return the home path for local iRODS resources """
         return settings.IRODS_CWD
```

Comment:
The reason for this so that someone doesn't have to remember that settings.IRODS_CWD is needed. This documents that. The reason that it is public is that this is required in order to implement this code as a mixin; private code would not be seen in hs_core/models.py:AbstractResource.